examples/markdown/README.md

```diff
@@ -0,0 +1,11 @@
+## Markdown example
+
+Intended as a demo of using [El::from_markdown()](https://docs.rs/seed/latest/seed/virtual_dom/node/el/struct.El.html#method.from_markdown) and ```md!``` for markdown conversion.
```

Comment:
Why 3 backticks? (` ```md!``` `)

Reply:
I intended to display it as code.

Reply:
Just realized it's only one backtick

Reply:
One backtick for inline code, three for multi-line. There are often some differences in rendering - e.g. one backtick => `<pre>xx</pre>`, 3 backticks => `<pre><code>xx</code></pre>`.